examples/bri-3/src/bri/auth/agents/authAccessor.agent.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../../../../prisma/prisma.service';
+
+@Injectable()
+export class AuthAccesorService extends PrismaService {
```

Comment:
i dont think new service is needed, this looks like it should be part of bpi subject storage agent
authAccessor.findOne doesnt say anything about bpi subject being returned

Reply:
I wanted to keep auth related code inside `auth` folder, and leave other CRUD operations inside identity. I don't this change is causing any blockers